lib/src/collection/treeset.dart

```diff
@@ -438,7 +436,7 @@ class AvlTreeSet<V> extends TreeSet<V> {
   }
 
   bool remove(Object item) {
-    AvlNode<V> x = _getNode(item);
+    AvlNode<V> x = _getNode(item as V);
```

Comment:
Actually, nvm, we _are_ implementing `Iterable`. In that case we can probably do this:

```dart
if (item is! V) {
  return false;
}

...
```